src/platforms/common/usage/sdk-fingerprinting.mdx

```diff
@@ -9,6 +9,8 @@ notSupported:
   - native.wasm
   - native.minidump
   - unreal
+  - rust
```

Comment:
events have a fingerprint in rust, and you can mess with that in `before_send`: https://docs.rs/sentry/latest/sentry/types/protocol/v7/struct.Event.html#structfield.fingerprint